static/css/styles.css

```diff
@@ -100,21 +101,57 @@ input {
     padding: 10px 30px;
 }
 
-.primary-content {
-    float: left;
+.helping-text {
+    margin: 0 5px 5px 0;
+    text-align: right;
 }
 
-@media only screen and (min-width: 801px) {
-.primary-content div {
-    float: left;
-    line-height: 36px;
-    width: 200px;
+.entry-container {
+    display: flex;
+    align-items: center;
 }
 
-div.question-content {
+.entry-item {
+    flex-shrink: 0;
+    margin: 0 5px;
+}
+
+.entry-text {
+    flex-shrink: 1;
+}
+
+.entry-name {
+    width: 100px;
```

Comment:
I think I had this at 200px before... at 100px, it usually isn't wide enough to fit the name and Andrew ID on the same line.

Reply:
I can increase to about 130px or so (which allows at least both of our names to fit on the same line, unlike before), but if I go longer, it pushes the buttons out of the bounding box on iPhone X. wdyt?

Reply:
Can we do 200px and then shorten it to 130 below 800 width?

Reply:
Yep, done.